coursedata/level-defaults/nl.yaml

```diff
@@ -171,7 +171,7 @@
 5:
     start_code: "print '5 keer 5 is ' 5 * 5"
     intro_text: |
-        `ask`, `print`, `if` en `repeat` werken nog zoals in level 4.
+        `ask`, `print` en `if` werken nog zoals in level 4.
```

Comment:
Nice catch! This same error occurs in all others langs too :( Could you fix these too, or f that is too much work, make an issue so I don't forgot to fix it?